pkg/postgres/schema/node_components_cves_edges.go

```diff
@@ -0,0 +1,70 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"fmt"
+	"reflect"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+var (
+	// CreateTableNodeComponentsCvesEdgesStmt holds the create statement for table `node_components_cves_edges`.
+	CreateTableNodeComponentsCvesEdgesStmt = &postgres.CreateStmts{
+		Table: `
+               create table if not exists node_components_cves_edges (
+                   Id varchar,
+                   IsFixable bool,
+                   FixedBy varchar,
```

Comment:
Can we fix this redundancy? FixedBy != null is IsFixable

Reply:
Such tasks should not go under postgres umbrella. It is more of a deprecation that requires us to visit UI and service layer, wrapper code, etc.